cilium-docker-plugin.Dockerfile

```diff
@@ -3,8 +3,7 @@ LABEL maintainer="maintainer@cilium.io"
 ADD . /go/src/github.com/cilium/cilium
 WORKDIR /go/src/github.com/cilium/cilium/plugins/cilium-docker
 ARG LOCKDEBUG
-ARG V
-RUN make CGO_ENABLED=0 GOOS=linux LOCKDEBUG=$LOCKDEBUG PKG_BUILD=1 EXTRA_GOBUILD_FLAGS="-a -installsuffix cgo"
```

Comment:
`PKG_BUILD` used to be here, and you don't explicitly call it out here, but there's a few Makefiles that still check it, I believe to skip unnecessary builds in these container runs. Was this intentional?

Reply:
I should update commit message, I removed `PKG_BUILD` as the particular Makefiles didn't check it.

Reply:
Oh, I assumed the make ran back up at the top directory but now that I look I see the workdir contradicts this. Makes sense, thanks for taking a look :-)